edalize/flows/vpr.py

```diff
@@ -0,0 +1,31 @@
+# Copyright edalize contributors
+# Licensed under the 2-Clause BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-2-Clause
+
+import os.path
+
+from edalize.flows.edaflow import Edaflow
+
+
+class Vpr(Edaflow):
+    """VPR is an open source academic CAD tool designed for the exploration of new FPGA architectures and CAD algorithms, at the packing, placement and routing phases of the CAD flow"""
+
+    argtypes = ["vlogdefine", "vlogparam"]
+
+    FLOW = [
+        ("yosys", ["vpr"], {"output_format": "blif"}),
+        ("vpr", [], {}),
+    ]
+
+    FLOW_OPTIONS = {}
+
+    def __init__(self, edam, work_root, verbose=False):
+        Edaflow.__init__(self, edam, work_root, verbose)
+
+    def build_tool_graph(self):
```

Comment:
I believe this function can be dropped since it just calls the parent class

Reply:
fixed